### PR TITLE
made changes to a statement

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
The statement I made changes to was due to the fact that explicit values were used in the link generator method but no ambient value was used

### CODE REVIEW

**Comment by sammychinedu2ky** [comment]
@rynowak @serpent5

File: aspnetcore/fundamentals/routing.md [hunk 1]
```diff
@@ -687,7 +687,7 @@ The preceding code:
 * Returns `/Widget/Index/17`
 * Gets <xref:Microsoft.AspNetCore.Routing.LinkGenerator> via [DI](xref:fundamentals/dependency-injection).
 
-The following code provides no ambient values and explicit values:
+The following code provides no ambient values but only explicit values:
```
Comment (wadepickett) [inline aspnetcore/fundamentals/routing.md:690]:
```suggestion
The following code only provides only explicit values and no ambient values:
```

**Comment by wadepickett** [comment]
Thanks @sammychinedu2ky

**Comment by sammychinedu2ky** [comment]
You're welcome 😅

**Pull request merged**